packages/build/tests/core/snapshots/tests.js.md

```diff
@@ -42,14 +44,15 @@ Generated by [AVA](https://avajs.dev).
       --branch          Repository branch.␊
                         Default: automatically guessed                      [string]␊
       --dry, --dry-run  Run in dry mode, i.e. printing steps without executing them.␊
+    ␊
                         Default: false                                     [boolean]␊
-      --nodePath        Path to the Node.js binary to use in the build command and␊
-                        plugins.␊
+      --nodePath        Path to the Node.js binary to use in the build command and p␊
```

Comment:
Just to confirm, is this normal? Not familiar with ava and its snapshots.

Reply:
This is not a problem with Ava snapshots, but with Yargs. The latest version of Yargs seem to have some bug related to how wrapping is done with the `--help` output. The test snapshot is reflecting that change of behavior from Yargs.

Reply:
Oh, good to know. I don't suppose this is fixed in v17? There's not any pressure to update ofc, the important thing is to use the same version across all packages :)